_security/authentication-backends/saml.md

```diff
@@ -19,37 +19,35 @@ This profile is meant for use with web browsers. It is not a general-purpose way
-
-   You can remove "customize" from the file names if you plan to modify and keep these files for production.
-   {: .tip }  
-
-1. In the `docker-compose.yml` file, specify your OpenSearch version number in the `image` field for nodes 1 and 2 and the OpenSearch Dashboards server. For example, if you are running OpenSearch version {{site.opensearch_major_minor_version}}, the `image` fields will resemble the following examples:
-   
-   ```yml
-   opensearch-saml-node1:
-    image: opensearchproject/opensearch:{{site.opensearch_major_minor_version}}
-    ```
-    ```yml
-    opensearch-saml-node2:
-    image: opensearchproject/opensearch:{{site.opensearch_major_minor_version}}
-    ```
-    ```yml
-    opensearch-saml-dashboards:
-    image: opensearchproject/opensearch-dashboards:{{site.opensearch_major_minor_version}}
-    ```
-
-1. Access OpenSearch Dashboards at [http://localhost:5601](http://localhost:5601){:target='\_blank'}. Note that OpenSearch Dashboards immediately redirects you to the SAML login page.
-
-1. Log in to OpenSearch Dashboards. The default username is `admin` and the default password is set in your `customize-docker-compose.yml` file in the `OPENSEARCH_INITIAL_ADMIN_PASSWORD=<custom-admin-password>` setting.
-
-1. After logging in, note that your user in the upper-right is `SAMLAdmin`, as defined in `/var/www/simplesamlphp/config/authsources.php` of the SAML server.
+1. Visit the [saml-demo branch](https://github.com/opensearch-project/demos/tree/saml-demo) of the demos repository and download it to a folder of your choice. If you're not familiar with how to use GitHub, check out the [OpenSearch onboarding guide](https://github.com/opensearch-project/demos/blob/main/ONBOARDING.md) for helpful instructions.
+
+1. Navigate to the `demo` folder:
+   ```zsh
+   $ cd <path-to-demos-folder>/demo
+   ```
+
+1. Review the following files, as needed:
+
+   * `.env`: 
+     * Defines the OpenSearch and OpenSearch Dashboards version to use. The default is the latest version ({{site.opensearch_major_minor_version}}).
+     * Defines the `OPENSEARCH_INITIAL_ADMIN_PASSWORD` variable required by versions 2.12 and later.
+   * `./custom-config/opensearch_dashboards.yml`: Includes the SAML settings for the default `opensearch_dashboards.yml` file.
+   * `./custom-config/config.yml`: Configures SAML for authentication.
+   * `docker-compose.yml`: Defines an OpenSearch server node, an OpenSearch Dashboards server node, and a SAML server node.
+   * `./saml/config/authsources.php`: Contains the list of users that can be authenticated by this SAML domain.
+
+1. At the command line, run:
+   ```zsh
+   $ docker-compose up.
+   ```
+
+1. Access OpenSearch Dashboards at [http://localhost:5601](http://localhost:5601){:target='\_blank'}.
+
+1. Select `Log in with single sign-on`. This redirects you to the SAML login page.
+
+1. Log in to OpenSearch Dashboards with a user defined in `./saml/config/authsources.php` (such as, `user1` and `user1pass`).
```

Comment:
In the parenthetical, should "and" be "or"?

Reply:
no, its a pair of username and password. I've updated the text accordingly.